src/components/MainArea.vue

```diff
@@ -83,6 +83,7 @@ console.log(sportsWithEventsComputed)
         <span class="header">
           <component
             :is="sport.icon.comp"
+            :class="{ 'dark-icon' : sport.icon.name != 'nfl' }"
             class="icon"
```

Comment:
You can combine your class items here as:

`:class=['icon', { 'dark-icon' : sport.icon.name != 'nfl' }]`

Reply:
I actually had to do it in quotes too, like this:

:class="['icon', { 'dark-icon' : sport.icon.name != 'nfl' }]"